src/Config.php

```diff
@@ -297,7 +297,7 @@ public function routes() : array
         if ($this->inBuild()) {
             throw new BuildTimeVariableAccessException('Routes are not available during the build phase.');
         }
-        if (empty($this->variablesDef)) {
+        if (empty($this->routesDef)) {
```

Comment:
This should be flagged as a big issue. Our site went down due to this oops. We're now resolved, but updating to a fix version is important.